Tools/Scripts/generate-compile-commands

```diff
@@ -85,9 +85,10 @@ for j_file_name in glob.glob(compile_commands_dir + "/*.json"):
         continue
 
     # Try to load JSON File
```

Comment:
I would move this comment after updating the json contents.

Reply:
Nice, done.